src/Nox.Lib/Messaging/OutboxRepository.cs

```diff
@@ -11,40 +14,70 @@ internal class OutboxRepository : IOutboxRepository
         private readonly IPublishEndpoint _bus;
         private readonly ILogger<OutboxRepository> _logger;
         private readonly IUserProvider _userProvider;
+        private readonly NoxSolution _noxSolution;
+        private readonly IWebHostEnvironment _webHostEnvironment;
 
-        public OutboxRepository(IPublishEndpoint bus, ILogger<OutboxRepository> logger, IUserProvider userProvider)
+        public OutboxRepository(
+            IPublishEndpoint bus,
+            ILogger<OutboxRepository> logger,
+            IUserProvider userProvider,
+            IWebHostEnvironment webHostEnvironment,
+            NoxSolution noxSolution)
         {
             _bus = bus;
             _logger = logger;
             _userProvider = userProvider;
+            _noxSolution = noxSolution;
+            _webHostEnvironment = webHostEnvironment;
         }
+
         public async Task AddAsync<T>(T integrationEvent) where T : IIntegrationEvent
         {
             _logger.LogInformation($"Publish message {typeof(T)} to {_bus.GetType()}");
 
-            var cloudEventRecord =  new NoxMessageRecord<T>(integrationEvent);
-            await _bus.Publish(cloudEventRecord,            
+            var prefix = _webHostEnvironment.EnvironmentName == "Production" ?
```

Comment:
you case use this instead of hardcoded string:
![image](https://github.com/NoxOrg/Nox.Generator/assets/8514193/c4c582cd-d3b8-492c-83d3-f3eaff924a03)